### PR TITLE
FEAT: Support for Individual PDF (requires sphinxcontrib-jupyter=0.5.0)

### PR DESCRIPTION
**Pull request by mmcky**
This PR adds support for individual page PDF construction. 

- [x] requires `sphinxcontrib-jupyter=0.5.0` https://github.com/QuantEcon/sphinxcontrib-jupyter/pull/211

### CODE REVIEW

**Comment by mmcky** [comment]
- [ ] resolve following issue with compilation @AakashGfude. Looks like we need to check index dedent in `translate_all`

```bash
# Docutils version: 0.14 
# Jinja2 version: 2.10.1
# Last messages:
#   writing output... [ 46%] more_julia/data_statistical_packages
#   writing output... [ 48%] more_julia/general_packages
#   writing output... [ 49%] more_julia/generic_programming
#   writing output... [ 50%] more_julia/index
#   writing output... [ 51%] more_julia/need_for_speed
#   writing output... [ 53%] more_julia/optimization_solver_packages
#   writing output... [ 54%] more_julia/testing
#   writing output... [ 55%] more_julia/tools_editors
#   writing output... [ 57%] more_julia/version_control
#   writing output... [ 58%] multi_agent_models/aiyagari
# Loaded extensions:
#   sphinx.ext.mathjax (2.1.2) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/ext/mathja
x.py
#   sphinxcontrib.applehelp (1.0.1) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinxcontri
b/applehelp/__init__.py
#   sphinxcontrib.devhelp (1.0.1) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinxcontrib/
devhelp/__init__.py
#   sphinxcontrib.htmlhelp (1.0.2) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinxcontrib
/htmlhelp/__init__.py
#   sphinxcontrib.serializinghtml (1.1.3) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx
contrib/serializinghtml/__init__.py
#   sphinxcontrib.qthelp (1.0.2) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinxcontrib/q
thelp/__init__.py
#   alabaster (0.7.12) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/alabaster/__init__.py
#   sphinxcontrib.bibtex (unknown version) from /home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphin
xcontrib/bibtex/__init__.py
#   IPython.sphinxext.ipython_console_highlighting (unknown version) from /home/qebuild/anaconda3/envs/latest/lib/pyt
hon3.7/site-packages/IPython/sphinxext/ipython_console_highlighting.py
#   sphinxcontrib.jupyter (19.1.1) from /home/qebuild/repos/sphinxcontrib-jupyter/sphinxcontrib/jupyter/__init__.py
Traceback (most recent call last):
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/cmd/build.py", line 284, in build_main
    app.build(args.force_all, filenames)
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/application.py", line 345, in build
    self.builder.build_update()
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 319, in bu
ild_update
    len(to_build))
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 382, in bu
ild
    self.write(docnames, list(updated_docnames), method)
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 562, in wr
ite
    self._write_serial(sorted(docnames))
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/sphinx/builders/__init__.py", line 573, in _w
rite_serial
    self.write_doc(docname, doctree)
  File "/home/qebuild/repos/sphinxcontrib-jupyter/sphinxcontrib/jupyter/builders/jupyterpdf.py", line 132, in write_d
oc
    self.writer.write(doctree, destination)
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/writers/__init__.py", line 80, in wr
ite
    self.translate()
  File "/home/qebuild/repos/sphinxcontrib-jupyter/sphinxcontrib/jupyter/writers/jupyter.py", line 23, in translate
    self.document.walkabout(visitor)
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  [Previous line repeated 1 more time]
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/nodes.py", line 187, in walkabout
    visitor.dispatch_departure(self)
  File "/home/qebuild/anaconda3/envs/latest/lib/python3.7/site-packages/docutils/nodes.py", line 1895, in dispatch_de
parture
    return method(node)
  File "/home/qebuild/repos/sphinxcontrib-jupyter/sphinxcontrib/jupyter/writers/translate_all.py", line 686, in depar
t_list_item
    list_item_start = self.list_item_starts.pop()
IndexError: pop from empty list
```

**Comment by arnavs** [comment]
Added @AnjuJoon as a reviewer so she can merge if need be.

**Comment by arnavs** [comment]
I'm seeing errors with `FFMPEG` that are wrecking execution on nearly every lecture... but we don't see this on the actual site/jupyter. So something to do with PDF and `GR`? Dunno.

**Comment by arnavs** [comment]
(If this persists after we try it again, we can do what Simon suggests [here](https://github.com/JuliaIO/FFMPEG.jl/issues/12) 10 days ago.)

File: theme/templates/latex.tpl [hunk 1]
```diff
@@ -173,10 +174,10 @@
 
 ((* block maketitle *))
 
+\title{((( nb.metadata.get("latex_metadata", {}).get("title", "") | escape_latex )))}
+
 ((*- if nb.metadata.get("latex_metadata", {}).get("author", ""): -*))
 \author{((( nb.metadata["latex_metadata"]["author"] )))}
-((*- else -*))
-\author{Al Pacino}
```
Comment (arnavs) [inline theme/templates/latex.tpl]: @AakashGfude Can you elaborate on this change? I think we should credit Al Pacino for his contributions to the theme, if appropriate `:-P`

**Pull request merged**